builder/docker/communicator.go

```diff
@@ -194,8 +194,34 @@ func (c *Communicator) UploadDir(dst string, src string, exclude []string) error
 	return nil
 }
 
+// Download pulls a file out of a container using `docker cp`. We have a source
```

Comment:
@vtolstov `Upload` and `UploadDir` are already implemented for docker. `DownloadDir` is actually not part of the communicator interface. It seems like it should be and I agree it would be useful, though.